Presentation.Web/app/components/org/structure/org-structure.controller.ts

```diff
@@ -627,14 +627,14 @@
                                     if (rootNodes.length < 1)
                                         return;
                                     const rootUnit = rootNodes[0];
```

Comment:
selection of parent unit should also have this decoration - not just the one in the migration